src/keygen/keyshare.rs

```diff
@@ -19,17 +24,26 @@ use zeroize::ZeroizeOnDrop;
 /// This type must be stored securely by the calling application.
 #[derive(Clone, ZeroizeOnDrop)]
 pub struct KeySharePrivate {
-    pub(crate) x: BigNumber, // in the range [1, q)
+    x: BigNumber, // in the range [1, q)
 }
-
 impl Debug for KeySharePrivate {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         f.write_str("KeySharePrivate([redacted])")
     }
 }
-
 impl KeySharePrivate {
-    // Computes the "raw" curve point corresponding to this private key.
+    /// Get the private key share of the participant.
+    pub(crate) fn x(&self) -> &BigNumber {
+        &self.x
+    }
```

Comment:
suggestion: update naming here to match whatever you choose in `KeySharePublic`. e.g. Either do `as_bignumber` or implement the `AsRef<BigNumber>` trait.